data/profiles/install-centos.ipxe

```diff
@@ -3,5 +3,10 @@ set base-url <%=repo%>/images/pxeboot
 set params initrd=initrd.img ks=<%=installScriptUri%> hostname=<%=hostname%> ksdevice=bootif BOOTIF=01-${netX/mac} console=<%=comport%>,115200n8 console=tty0
 kernel ${base-url}/vmlinuz repo=<%=repo%> ${params}
 initrd ${base-url}/initrd.img
+
+# Below two lines are designed for progress info, downloading action itself is meaningless.
+initrd http://<%=server%>:<%=port%>/api/current/notification/<%=taskId%>/<%=steps%>
```

Comment:
would it avoid confusion if renaming "steps" to some phrase like "totalSteps"? It needs the variable in on-tasks to change as well.